app/components/DosesDescriptionList/DosesDescriptionList.module.scss

```diff
@@ -1,4 +1,5 @@
 @import "../../styles/vars.scss";
+@import "../../styles/typography.scss";
```

Comment:
mentioned in the description but I can remove the changes from this file if that's preferred

Reply:
Na this is all good! I can rebase ;)